testData/solutions/web.json

```diff
@@ -164,5 +164,131 @@
+                                }
+                            }
+                        }
+                    },
+                    "links": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/linkAdaptation",
+                            "options": {
+                                "emphasizeLinks": {
+                                    "outputValue": true
+                                }
+                            }
+                        }
+                    },
+                    "contrastTheme": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
+                            "conditionPath": "http://registry\\.gpii\\.net/common/highContrastEnabled",
+                            "true": {
+                                "transform": {
+                                    "type": "fluid.transforms.valueMapper",
+                                    "inputPath": "http://registry\\.gpii\\.net/common/highContrastTheme",
+                                    "defaultInputValue": "default",
+                                    "options": {
+                                        "black-white": {
+                                            "outputValue": "bw"
+                                        },
+                                        "white-black": {
+                                            "outputValue": "wb"
+                                        },
+                                        "black-yellow": {
+                                            "outputValue": "by"
+                                        },
+                                        "yellow-black": {
+                                            "outputValue": "yb"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "signLanguageEnabled": "http://registry\\.gpii\\.net/common/signLanguageEnabled",
+                    "signLanguage": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/signLanguage",
+                            "options": {
+                                "ase": {
+                                    "outputValue": "ase"
+                                },
+                                "gsg": {
+                                    "outputValue": "gsg"
+                                },
+                                "ils": {
+                                    "outputValue": "ils"
+                                }
+                            }
+                        }
+                    },
```

Comment:
If the value range for the signLanguage common term and the signLanguage OLB is the same (which it indeed looks like), you should be able to replace the above transformation with 

"signLanguage": "http://registry\\.gpii\\.net/common/signLanguage",

The valueMapper is only used if the valuerange of the two is different